package NBassUtProtos;

message TTestUsers {
    required string login = 1;
    optional uint64 release_at = 2;
    optional string client_ip = 3;
    optional string password = 4;
    optional string token = 5;
    optional string uuid = 6;
};

message TTestUsersTags {
    required string login = 1;
    required string tag = 2;
};
